CONTRIBUTING.md

```diff
@@ -96,58 +96,85 @@ there are other options.
 
 1.  Create a conda environment.
 
-It is good practice to maintain different environments for different versions
-of JWST and its dependencies. You will likely want to maintain one, for example,
-for the latest released version of JWST (i.e. what you get by doing `pip install jwst`),
-as well as one for development. Assuming the user has conda [installed](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html),
-here we will create a new conda environment called 'jwst_dev' where we can install the new branch of our cloned repository.
-
-	>> conda create -n jwst_dev python
-
-
-Doing this will create a new environment with just some basic packages
-(i.e setuptools, pip) installed.
-
-2. Installing `jwst` in this environment
-
-Make sure you are in your new environment:
-
-	>> conda activate jwst_dev
-
-And now in the top level of your local `jwst` repository, ensuring you're
-on the 'my_feature' branch:
-
-	>> pip install -e .
-This will install `jwst` from this cloned source code in 'editable' mode,
-meaning that you can import the code from this directory when within a Python
-session. This makes it easier for development because you can have the code
-you're editing somewhere convenient in your file system vs. with other packages
-in 'site-packages'. If you cloned the repository on your Desktop, for example,
-you can modify it there and Python will know that is where the source code is
-when you're importing it within a Python session.
-
-*Note* : If you use it, make sure to install iPython in your new environment
-as well. Otherwise, it will pick up packages from the base environment instead.
+    It is good practice to maintain different environments for different versions
+    of JWST and its dependencies. You will likely want to maintain one, for example,
+    for the latest released version of JWST (i.e. what you get by doing `pip install jwst`),
+    as well as one for development. Assuming the user has conda [installed](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html),
+    here we will create a new conda environment called 'jwst_dev' where we can install the new branch of our cloned repository.
+    
+    	>> conda create -n jwst_dev python
+    
+    
+    Doing this will create a new environment with just some basic packages
+    (i.e setuptools, pip) installed.
+
+2. Install `jwst` in this environment
+
+    Make sure you are in your new environment:
+    
+    	>> conda activate jwst_dev
+    
+    And now in the top level of your local `jwst` repository, ensuring you're
+    on the 'my_feature' branch:
+    
+    	>> pip install -e .[contrib]
```

Comment:
```suggestion
    	>> pip install -e ".[contrib]"
```
I think some shells don't like the bare `[]` (this suggestion also makes the command match the other examples which install with a dependency group).